card/mod/date/set/self/script_datepicker.rb

```diff
@@ -0,0 +1,6 @@
+include_set Abstract::VendorCodeFile
+Self::ScriptMods.add_item :script_datepicker
+
+def source_files
+  %w[moment/min/moment-with-locales.min.js tempusdominus/build/js/tempusdominus-bootstrap-4.js]
```

Comment:
I replaced the jquery-ui calendar with a bootstrap 4 solution

Reply:
👍